packages/localizations/src/pt-BR.ts

```diff
@@ -0,0 +1,546 @@
+import { DeepRequired, LocalizationResource } from '@clerk/types';
+
+const commonTexts = {
+  signIn: {
+    phoneCode: {
+      title: 'Verifique seu celular',
+      subtitle: 'para continuar em {{applicationName}}',
+      formTitle: 'Código de verificação',
+      formSubtitle: 'Insira o código enviado para seu celular',
+      resendButton: 'Reenviar código',
+    },
+  },
+} as const;
+
+export const enUS: DeepRequired<LocalizationResource> = {
```

Comment:
```suggestion
export const ptBR: DeepRequired<LocalizationResource> = {
```

Reply:
We also need to export this from `./index.ts`. Happy to do it myself but I don't want to hijack your PR :)

Thanks for the contribution!